pkg/bpf/bpf_linux.go

```diff
@@ -278,6 +293,8 @@ func ObjGet(pathname string) (int, error) {
 		uintptr(unsafe.Pointer(&uba)),
 		unsafe.Sizeof(uba),
 	)
+	runtime.KeepAlive(&pathStr)
```

Comment:
`pathStr` is already a pointer, so this should be `runtime.KeepAlive(pathStr)`.

Reply:
yep thanks. Its interesting that doesn't segfault somehow in CI or throw any warnings anywhere. I guess its a valid address, also wonder if there is a linter that could throw a warning if unsafe pointers are passed to syscalls and _not_ also passed into KeepAlive. Doesn't seem like too much work to catch that pattern. Anyways will fix thanks again.

Reply:
I haven't found any static analysis tool doing that. Maybe the upcoming the `checkptr` feature in Go 1.14 will do that (haven't checked yet).